pkg/operator/ceph/object/spec.go

```diff
@@ -99,7 +99,10 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) v1.PodTemplateSpec
 					}}}}
 		podSpec.Volumes = append(podSpec.Volumes, certVol)
 	}
-	preferredDuringScheduling := false
+
+	// Fallback pod anti affinity for rgw pods to PreferredDuringSchedulingIgnoredDuringExecution
```

Comment:
How about these comments?
```suggestion
	// If host networking is enabled, required pod anti-affinity is added to the rgw daemons due to the potential port conflict
```